### PR TITLE
Adding support for another type (as)

### PR DESCRIPTION
**Pull request by pruzanov**

### CODE REVIEW

File: internal/supported_types.py [hunk 1]
```diff
@@ -12,7 +12,25 @@
 
 
 def get_supported():
-    return ['qi', 'b', 'i', 's', 'algebraic', 'msas', 'object']
+    return ['qi', 'b', 'i', 's', 'as', 'algebraic', 'msas', 'object']
```
Comment (pruzanov) [inline internal/supported_types.py:15]: Main changes here - added 'as' type

File: internal/supported_types.py [hunk 2]
```diff
@@ -12,7 +12,25 @@
...
+"""
+   Flatten nested array and return string in array if needed
+   This function handles 'as' type at present but should go into AS class
+   eventually. Also, other classes may use validate_value() function
+   which would ensure correct formatting of the dict structure
+   that goes into project_config_info.jsonfile
+"""
+
+
+def flatten(entry):
```
Comment (pruzanov) [inline internal/supported_types.py:27]: this is a support function

File: internal/supported_types.py [hunk 3]
```diff
@@ -127,6 +150,28 @@ def get_default():
         return "Enter value"
 
 
+class AS:
```
Comment (pruzanov) [inline internal/supported_types.py:153]: Added class for the new type

Comment (pruzanov) [reply internal/supported_types.py:153]: Other changes are cosmetic

**Review by lheisler — APPROVED** [review]

**Pull request merged**